resqpy/model.py

```diff
@@ -3274,11 +3274,9 @@ def iter_trajectories(self):
       """
       import resqpy.well  # Imported here for speed, module is not always needed
 
-      parts = self.parts_list_of_type('WellboreTrajectoryRepresentation')
-      for part in parts:
-         traj_root = self.root_for_part(part)
-         traj = resqpy.well.Trajectory(self, trajectory_root = traj_root)
-         yield traj
+      uuids = self.uuids(obj_type = "WellboreTrajectoryRepresentation")
+      for uuid in uuids:
+         yield resqpy.well.Trajectory(self, uuid = uuid)
```

Comment:
Out of curiosity, why are we switching from using parts to uuids?

Reply:
This relates to issue #17 , our longer-term goal to clean up the code base. Presently, uuids roots and part names are used interchangably as the "key" to reference objects, which leads to a fair amout of messy code having to covert between them all. We decided to standardise on `uuids`, as use that as the only "key" if possible throughout the codebase.

Most classes now support an init method with a `uuid` parameter.